Code/Framework/AzToolsFramework/Tests/Prefab/PrefabUndoComponentPropertyTestFixture.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Prefab/PrefabUndoComponentPropertyTestFixture.h>
+
+#include <AzCore/DOM/Backends/JSON/JsonSerializationUtils.h>
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <AzToolsFramework/Prefab/PrefabDomUtils.h>
+#include <AzToolsFramework/ToolsComponents/TransformComponent.h>
+
+namespace UnitTest
+{
+    void PrefabUndoComponentPropertyTestFixture::SetUpEditorFixtureImpl()
+    {
+        PrefabTestFixture::SetUpEditorFixtureImpl();
+
+        m_prefabOverridePublicInterface = AZ::Interface<PrefabOverridePublicInterface>::Get();
+        ASSERT_TRUE(m_prefabOverridePublicInterface);
+    }
+
+    bool PrefabUndoComponentPropertyTestFixture::CreateWheelEntityHierarchy(EntityInfo& wheelEntityInfo)
+    {
+        AZ::EntityId wheelEntityId = CreateEditorEntityUnderRoot(WheelEntityName);
+        EntityAlias wheelEntityAlias = FindEntityAliasInInstance(GetRootContainerEntityId(), WheelEntityName);
+        EXPECT_FALSE(wheelEntityAlias.empty());
```

Comment:
Perhaps ASSERT_FALSE/ASSERT_TRUE would be more appropriate for setup related calls?

Reply:
Yeah. I should not assert on the return type (bool); instead, I should use all assertions in the setup calls.